Hi <%= @user.login %>,

<%= @o.planned ? 'a planned' : 'an unplanned' %> outage has been reported and <%= @vpses.count %> of your VPS <%= @vpses.count > 1 ? 'are' : 'is' %> affected.

Affected VPS:

<% @vpses.each do |vps| -%>
  - VPS #<%= vps.vps_id %> - <%= vps.vps.hostname %> (<%= vps.node.domain_name %>)
<% end -%>

Date and time: <%= @o.begins_at.localtime.strftime('%Y-%m-%d %H:%M %Z') %>
Expected duration: <%= @o.duration %> minutes
Affected systems: <%= @o.outage_entities.map { |e| e.real_name }.join(', ') %>
Outage type: <%= @o.outage_type %>
Reason: <%= @o.en_summary %>
Handled by: <%= @o.outage_handlers.map { |h| h.user.full_name }.join(', ') %>
<% if @o.en_description && !@o.en_description.empty? -%>

<%= @o.en_description %>
<% end -%>

Best regards,

vpsAdmin
